app/Http/Controllers/ItemsController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Items;
+
+class ItemsController extends Controller
+{
+    public static function show() 
```

Comment:
Variable heet test, is niet netjes :)

Reply:
veranderd boven de review, ben sneller dan jij